test/Passes/pass-vectorization.mlir

```diff
@@ -0,0 +1,64 @@
+// RUN: tpp-opt %s --tile-consumer-and-fuse-producers -bufferize --vectorization-pass --split-input-file | FileCheck %s
+
+#map = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d2, d3, d5)>
+#map1 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d1, d2, d5, d4)>
+#map2 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d3, d4)>
+module {
+  func.func @entry(%arg0: tensor<4x4x4x4xf32>, %arg1: tensor<4x4x4x4xf32>, %arg2: tensor<4x4x4x4xf32>) -> tensor<4x4x4x4xf32> {
```

Comment:
I'd suggest using different sizes for each dimension for readability and it's easier to spot if everything works as intended.